isis/src/system/apps/isisdataeval/tools/README.md

```diff
@@ -63,53 +70,337 @@ Finally, by setting --ghostdir '$ISISDATA', this now provides a connection to
-isisdata_mockup.py --saveconfig --isisdata $ISISDATA/smart1   --outpath mockprocessing/isisdatamockup/smart1   --ghostdir '$ISISDATA/smart1'
-isisdata_mockup.py --saveconfig --isisdata $ISISDATA/voyager1 --outpath mockprocessing/isisdatamockup/voyager1 --ghostdir '$ISISDATA/voyager1'
+isisdata_mockup.py --saveconfig --hasher all --isisdata $ISISDATA/base     --outpath mockprocessing/isisdatamockup/base     --ghostdir '$ISISDATA/base'     --tojson isisdata_mockup_base.json
+isisdata_mockup.py --saveconfig --hasher all --isisdata $ISISDATA/hayabusa --outpath mockprocessing/isisdatamockup/hayabusa --ghostdir '$ISISDATA/hayabusa' --tojson isisdata_mockup_hayabusa.json
+isisdata_mockup.py --saveconfig --hasher all --isisdata $ISISDATA/smart1   --outpath mockprocessing/isisdatamockup/smart1   --ghostdir '$ISISDATA/smart1'   --tojson isisdata_mockup_smart1.json
+isisdata_mockup.py --saveconfig --hasher all --isisdata $ISISDATA/voyager1 --outpath mockprocessing/isisdatamockup/voyager1 --ghostdir '$ISISDATA/voyager1' --tojson isisdata_mockup_voyager1.json
 
 # Copy/install the desired files for the test
-rsync -av --files-from=isisdataeval_isisdata_mockup_files.lis mockprocessing/isisdatamockup/  mockup/
+rsync -av --files-from=isisdataeval_isisdata_mockup_files.lis mockprocessing/isisdatamockup/ mockup/
 /bin/rm -rf mockprocessing
 
-# Run an inventory test for the mockup
+# Run an inventory on the local ISISDATA mockup test data
 isisdataeval isisdata=mockup datadir=mockup toinventory=isisdata_mockup_inventory.csv toissues=isisdata_mockup_issues.csv toerrors=isisdata_mockup_errors.csv hash=md5
+```
+
+# Other products
+The `isisdata_mockup.py` application can also produce a summary procesing log file of the run used to create a mockup. It can also be ran standalone on any system to provide as an comparison dataset for any mockup. This command produces a summary of the full ISISDATA installation without producing the mockup in `--outpath` (it is now an optional parameter), computes only the md5 hash and writes the entire contents to the JSON output file specified in `--tojson`:
 
+```
+isisdata_mockup.py -v  --hasher=md5 --isisdata /opt/isis3/data --ghostdir '$ISISDATA' --tojson isisdata_full_md5.json
 ```
 
+This example provides the results of a much smaller directory, $ISISDATA/base/dems, that also computes the md5 hash, including all `*.db` and `*.conf` files, does not generate, unless given, any absolute paths, and **does not** produce the mockup directory:
+```
+isisdata_mockup.py --hasher md5 --isisdata /opt/isis3/data/base/dems --ghostdir '$ISISDATA/base/dems' --tojson isisdata_base_dem_md5.json
+```
 
+And here is the contents of the output JSON file, `isisdata_base_dem_md5.json`:
+```
+{
+  "program": {
+    "name": "isisdata_mockup.py",
+    "version": "0.2",
+    "date": "2023-03-29",
+    "runtime": "2023-03-29T21:02:03.721127 UTC",
+    "endtime": "2023-03-29T21:02:39.887436 UTC",
+    "elapsedtime": "0:00:36.166309",
```

Comment:
could we replace this with a truncated version so it isn't as verbose?

Reply:
Done.